scripts/topiary

```diff
@@ -33,32 +33,26 @@ Example usage:
 from topiary.commandline_args import (
     arg_parser,
     rna_gene_expression_dict_from_args,
-    rna_transcript_expression_dict_from_args
+    rna_transcript_expression_dict_from_args,
+    write_outputs,
 )
 from topiary import predict_epitopes_from_args, epitopes_to_dataframe
 
 
-args = arg_parser.parse_args()
-
-def main():
-    print("Topiary commandline arguments:")
-    print(args)
+def main(args):
     epitopes = predict_epitopes_from_args(args)
     gene_expression_dict = rna_gene_expression_dict_from_args(args)
     transcript_expression_dict = rna_transcript_expression_dict_from_args(args)
     df = epitopes_to_dataframe(
         epitopes,
         gene_expression_dict=gene_expression_dict,
         transcript_expression_dict=transcript_expression_dict)
-    print("First 10 epitope predictions:")
-    print(df.ix[:10])
+    write_outputs(df, args)
     print("Total count: %d" % len(df))
-    if args.output_csv:
-        print("Saving %s..." % args.output_csv)
-        df.to_csv(args.output_csv, index=True, index_label="#")
-    if args.output_html:
-        print("Saving %s..." % args.output_html)
-        df.to_html(args.output_html, index=True)
 
 if __name__ == "__main__":
-    main()
+    args = arg_parser.parse_args()
+    print("Topiary commandline arguments:")
+    print(args)
```

Comment:
Yeah, I'm finding it handy to see how the commandline args got parsed. Do you think it's too noisy?